style.css

```diff
@@ -0,0 +1,171 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto:wght@300&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  scroll-behavior: smooth;
+  font-family: Roboto, sans-serif;
+}
+
+.class {
+  color: aqua;
+}
+
+.header {
+  position: fixed;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  height: 3rem;
+  background-color: #3c3a39;
+}
+
+.logo-nombre {
+  color: #ffff;
+  margin-left: 15px;
+}
+
+.hamburger {
+  background-image: url('./img/Icon\ -\ Menu.png');
+  width: 1rem;
+  height: 1rem;
+  margin-right: 15px;
+}
+
+.navlinks {
+  display: none;
+}
+
+.navlinks.active {
+  display: flex;
+}
+
+.principal {
+  /* Headline */
+
+  /* Auto layout */
+
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px;
+  gap: 12px;
+  position: static;
+  width: 100%;
+  height: 100vh;
+  left: 0;
+  top: 92px;
+
+  /* Theme 3 / Neutral 1 */
+
+  z-index: -1;
+  background: #1c1a19;
+  background-image: url(./img/illustration-header-mobile.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.tittle {
+  /* Primary Text */
+
+  width: 327px;
+  height: 124px;
+  font-family: 'Crete Round';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 56px;
+  line-height: 64px;
+
+  /* or 114% */
+
+  letter-spacing: 0.37px;
+
+  /* Theme 3 / Primary 1 */
+
+  color: #ff6b00;
+
+  /* Inside auto layout */
+
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.profession h2 {
+  width: 327px;
+  height: 27px;
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+
+  /* or 133% */
+
+  display: flex;
+  align-items: center;
+
+  /* Theme 3 / Primary 4 */
+
+  color: #fff5e1;
+
+  /* Inside auto layout */
+
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.text p {
+  /* Description */
+
+  width: 327px;
+  height: 123px;
+
+  /* Theme 1 / Body Reguler 2 */
+
+  font-family: 'Poppins';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+
+  /* or 150% */
+
+  /* Theme 3 / Neutral 11 */
+
+  color: #ffff;
+
+  /* Inside auto layout */
+
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
```

Comment:
- [ **OPTIONAL** ] I suggest you remove the unnecessary empty space and comments in all your code base to make it cleaner and readable.